### PR TITLE
Add task 'copy gitconfig' to common

### PR DESCRIPTION
**Pull request by runamoon**
https://gyazo.com/4906342b547c4d24e186db577ba7bb79

TODO
* [x] spec

### CODE REVIEW

File: spec/default/common_spec.rb [hunk 1]
```diff
@@ -11,3 +11,8 @@
 describe package('vim') do
   it { should be_installed }
 end
+
+describe file('/home/ubuntu/.gitconfig'), if: os[:family] == 'ubuntu' do
+  it { should be_owned_by 'ubuntu' }
+  it { should be_mode 600 }
```
Comment (ojiry) [inline spec/default/common_spec.rb]: Why is permission 600?

Comment (runamoon) [inline spec/default/common_spec.rb]: copy and paste!!

File: spec/default/common_spec.rb [hunk 2]
```diff
@@ -11,3 +11,8 @@
 describe package('vim') do
   it { should be_installed }
 end
+
+describe file('/home/ubuntu/.gitconfig'), if: os[:family] == 'ubuntu' do
```
Comment (ojiry) [inline spec/default/common_spec.rb]: I want to remove `if` because os is sure ubuntu.

File: spec/default/common_spec.rb (hunk 1 above)
Comment (ojiry) [inline spec/default/common_spec.rb]: My local .gitconfig is 755.
Could you please check default permission and change it.

**Comment by ojiry** [comment]
👍

**Pull request merged**